server/lib/notifications.js

```diff
@@ -286,15 +286,21 @@ async function notifyByEmail(activity) {
       notifyAdminsOfCollective(activity.data.collective.id, activity);
       break;
 
-    case activityType.COLLECTIVE_APPLY:
+    case activityType.COLLECTIVE_APPLY: {
+      const user = await models.User.findByPk(activity.data.user.id);
```

Comment:
The place where the activity is created doesn't seems to include the user id in the payload. But it does include the user email, so we can probably skip the DB query: https://github.com/opencollective/opencollective-api/blob/fde4e0b20e7d53dab447f2fa7f166c180ced57ea/server/models/Collective.js#L1510-L1548

Reply:
Done 👍